src/test/java/edu/ucla/library/prl/harvester/services/HarvestJobSchedulerServiceIT.java

```diff
@@ -137,9 +137,9 @@ public void afterEach(final Vertx aVertx, final VertxTestContext aContext) {
         wipeBackingServices.compose(nil -> {
             if (myService != null) {
                 return myService.close();
-            } else {
-                return Future.succeededFuture();
             }
+
+            return Future.succeededFuture();
```

Comment:
I'm ambivalent on this convention, I'm just used to doing it the other way and we don't have an automated check for it in the Maven build. Looks like [a new rule for it was proposed](https://github.com/checkstyle/checkstyle/issues/9698) but stalled for some reason.